Add Mapping and Modding Timeline article

pishifat opened the pull request:
re #3881 

i left sources for each point in comments but most are only accessible to moderators
also added info to BAT managers stub

---

In addition to the very detailed [timeline](https://osu.ppy.sh/wiki/Mapping_and_Modding_Timeline), this change adds a lot of mapping and modding history to many other parts of the wiki.

cl8n commented:
I'm not sure it's enough to put the "what" in this timeline without the "why", if it's to be used for the purpose described in linked issue. the "Extra relevant links/notes" column of the sheet already has a lot of stuff listed you could use

cl8n commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,302 @@
+
+### October
+
+- **2018-10-22:** [Ascendance](https://osu.ppy.sh/users/2931883) becomes a NAT leader after [Mao](https://osu.ppy.sh/users/2204515) steps down.
```
this type of thing is good to document as history ofc but I feel like it also needs to be connected to other points in this document or at least commented on why it's significant. (for this line) what did Ascendance do as NAT leader? did he have an impact on modding and mapping worth writing down? this doesn't tell reader anything important and it's not easy to discover answers to these questions if you weren't aroudn at the time

cl8n reviewed (dismissed):
some random thoughts skimming thru this once, will leave a grammar/wording review for later

I thought it was odd that most of this documents only the "official" responses that changed mapping and modding, and not developments of community opinion or anything. most notably there is none of the big mapping dramas talked about here, for a lot of people that's what defined entire eras of osu so it seems important

pishifat commented:
> I'm not sure it's enough to put the "what" in this timeline without the "why", if it's to be used for the purpose described in linked issue. the "Extra relevant links/notes" column of the sheet already has a lot of stuff listed you could use

added "why"s for everything that needed it (hopefully)

> I thought it was odd that most of this documents only the "official" responses that changed mapping and modding, and not developments of community opinion or anything. most notably there is none of the big mapping dramas talked about here, for a lot of people that's what defined entire eras of osu so it seems important

added mentions of Genryuu Kaiko, Last Journey Home, Asymmetry, and Rendezvous in places that are relevant or explanations for changes. there's a lot of drama about repeated issues and drama that amounted to nothing, so i dont think they'd make sense to include. if you have something in mind let me know

dorothy3242 commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,453 @@
+    - BN tiers were replaced with [Probationary Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators#probationary-beatmap-nominators). Two probationary BN members could not nominate the same beatmap and users could not be in Probation for more than two months at a time.
+    - Beatmap Nominators were given purple titles on the forums.
+    - [Okoratu](https://osu.ppy.sh/users/1623405) and [JBHyperion](https://osu.ppy.sh/users/4879508) were appointed as QAT leaders <!-- TODO: stub for QAT Leaders --> by a vote of BN/QAT members. Their responsibility was to work with the osu!team to maintain the mapping and modding communities. 
+    - *Elite Nominator* profile badges and forum titles were rewarded to exceptional Beatmap Nominators.
```
uhhhh, pretty sure this was at the end of 2019, I never heard of anything similar happening in 2017.

dorothy3242 commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,453 @@
+
+- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) released the Mapset Verifier <!-- TODO: stub --> tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
+- **2019-08-06:** Beatmap Nominators became able to participate in BN application evaluations.
+    - Their input was used to determine applicants and to judge potential new NAT additions. <!-- https://github.com/pishifat/qat/commit/a8245a4c6fefcfe9368ffc1eee7e13ad6f8bb6af -->
```
Hmm, but BNs' evaluations don't play a very large role in this, so saying that they determine applicants (? applicants what) seems misleading

dorothy3242 commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,453 @@
+- **2019-05-05:** The BN/QAT restructure was finalized.
+    - The Quality Assurance Team became the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
+    - Beatmap Nominators became responsible for disqualifications and veto mediation.
+    - The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks. <!-- https://osu.ppy.sh/home/news/2019-05-05-bn-and-qat-restructure -->
```
At this time, BN applications changed from being cycle based to allowing users to apply at any time, with individualized cooldown periods if they did not get in or were removed. This is a pretty big shift from the previous large and occasional waves being replaced with new bns being added in small amounts more regularly.

Besides saving the NAT from spreadsheet insanity, I think that's one of the bigger changes the website offered for the community view.

dorothy3242 commented on wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md:
```diff
@@ -0,0 +1,65 @@
+
+The United Beat-Knights of Ranking Criteira (*UBKRC*) was a new group with similar goals to the Ranking Criteria Council, however it was handled with less strictness. The group chose an intentionally stupid name to appear more friendly than its predecessor. The group was managed by [Okoratu](https://osu.ppy.sh/users/1623405) and [pishifat](https://osu.ppy.sh/users/3178418) until completion.
+
+Within the UBKRC, teams of experts were chosen to handle each area of the general Ranking Criteria and game-mode specific sections. Users could join if they showed an active interest in any group.
```
https://docs.google.com/spreadsheets/d/1PBF3XOLIWOScTxmSzzF1-fqjIaDhrfX3rPxcw6IAP5M/edit#gid=0 should have most or all of said people listed who worked on UBKRC. Feels like it should be listed if the RC council is.

Ephemeralis commented:
will wait for cl8n's review but it looks great to me, we def need to get this out sooner rather than later, it's an enormously valuable document

TicClick commented on wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md:
```diff
@@ -0,0 +1,162 @@
+# Ranking Criteria Council
+
+The **Ranking Criteria Council** was a group of users tasked with reworking the [Ranking Criteria](/wiki/Ranking_Criteria) in 2016.
```
make it a time span for clarity, something like "in 2016–2018", otherwise it sounds like they were only active in '16

pishifat replied:
but they were only active in 2016, new group below mentions 2016-2018

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+# Mapping and modding timeline
+
+Mapping and modding systems are constantly improving. The **Mapping and modding timeline** aims to document the biggest system changes and to serve as a reference tool when incorporating new changes.
```
I may be wrong, but "The timeline aims to serve as a reference when incorporating new changes" makes it sound like the timeline itself would make the changes.

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+    - Proto-bubbles were ignored by most BAT members, so the MAT had no purpose without this change.
+    - This gave MAT considerably more influence in the ranking process.<!-- https://osu.ppy.sh/community/forums/topics/38403 -->
+- **2011-10-10:** MAT members were no longer allowed to nominate or vote on new additions to their team.
+    - Having gained bubbling permissions, it was assumed that the MAT would vote with circlejerk-related reasons, so this was prevented before damage could be done.
```
uhh, I am sure we can do without referencing group masturbation here

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+    - The BAT (often called newBAT at this time) nominated beatmaps for rank.
+    - The QAT ensured qualified beatmaps were sufficient quality. Beatmaps that did not meet quality standards were disqualified.
+    - This essentially reversed the merging of the BAT and MAT last year.
+    - The split was implemented as part of a manual trial for [moddingv2](/wiki/Beatmap_Discussion) attempting to automate BAT promotion and retirement. <!-- https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat -->
```
```suggestion
    - The split was implemented as part of a manual trial for [modding v2](/wiki/Beatmap_Discussion) attempting to automate BAT promotion and retirement. <!-- https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat -->
```

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+### October
+
+- **2014-10-15:** The [Beatmap Management subforum](https://osu.ppy.sh/community/forums/115) replaced Trello.
+    - Public accessibility to the BAT increased, which was the goal of moddingv2.
```
I keep stumbling on it, is the spelling intentional?
```suggestion
    - Public accessibility to the BAT increased, which was the goal of modding v2.
```

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+    - The collective goal of this group was to rewrite all sections of the general Ranking Criteria. <!-- https://osu.ppy.sh/community/forums/topics/420229 -->
+- **2016-02-24:** Mapset and spread restructure was introduced to limit the number of difficulties in each difficulty tear per beatmap set.
+    - Negative public reception caused this to be rolled back immediately and its [forum thread](https://osu.ppy.sh/community/forums/topics/420223) accumulated 614 responses in 24 hours, making it osu!'s fastest moving thread.
+    - This attempt at undemocratic change caused the Ranking criteria council to immediately have a negative reputation. <!-- https://osu.ppy.sh/community/forums/topics/420223 -->
```
```suggestion
    - This attempt at undemocratic change caused the Ranking Criteria Council to immediately have a negative reputation. <!-- https://osu.ppy.sh/community/forums/topics/420223 -->
```

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+- **2016-08-05:** The [Quality Assurance Team blog](/wiki/Quality_Assurance_Team_Blog) was created.
+    - The QAT was receiving criticism about being too closed-off from the rest of the mapping community, so this blog was created to increase transparency for QAT decision making and community activity. <!-- https://osu.ppy.sh/community/forums/topics/578437 -->
+- **2016-08-26:** The [Code of Conduct](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) was implemented to establish modding behavior expectations.
+    - These rules and guidelines were treated as common sense up until this point. By documenting them, moderation could be more consistent.<!-- https://osu.ppy.sh/community/forums/topics/447557 -->
```
```suggestion
    - These rules and guidelines were treated as common sense up until this point. By documenting them, moderation could be made more consistent.<!-- https://osu.ppy.sh/community/forums/topics/447557 -->
```

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+### April
+
+- **2017-04-06:** [Beatmap Nominator Tiers](/wiki/Modding/Beatmap_Nominator_Tiers) were implemented, splitting the BN into two subgroups called "tiers".
+    - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering ranked.
```
```suggestion
    - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering Ranked.
```

TicClick commented on wiki/Mapping_and_Modding_Timeline/en.md:
```diff
@@ -0,0 +1,458 @@
+
+### April
+
+- **2020-04-11:** The Mappers' Report <!-- TODO: stub --> led by [Feerum](https://osu.ppy.sh/users/4815717) was created to summarize mapping community events via front page news posts.
```
if you have chance AND if there was anything significant in the sphere of modding and mapping, please document it

pishifat replied:
nothing's coming to mind

TicClick reviewed (approved):
looks good to me. if there aren't any unspoken agreement not to merge this before @cl8n's review, I can go for that
